### PR TITLE
Update to IL 19 from Highway63

### PR DESCRIPTION
**Pull request by jteresco**
Added many more waypoints.

### CODE REVIEW

**Comment by michihdeu** [comment]
sorry, I don't wanna deal with it.

**Comment by Markkos1992** [comment]
Unless it is obvious that a new alignment is included, I would say it is unnecessary.

Since the RDU meet is higher priority, I'll defer to @yakra.

**Comment by michihdeu** [comment]
The route was obviously through O'Hare Airport at `+X01 http://www.openstreetmap.org/?lat=41.957416&lon=-87.915773`. The route is realigned b/n York Rd and US12/45 now. It's just 0,22mi. Is it newsworthy? Likely. The additional wps are not newsworthy.

**Comment by rickmastfan67** [comment]
If the `I-490` point is to stay in the file, it should be marked as a 'hidden' point (just add the + in front of the label, no need to change the label itself) till the project is finished and it's opened up (2026?).

And I agree with @michihdeu that the reroute due to O'Hare expansion is worthy of an update entry.  However, it was only rerouted between York Rd & just east of Taft Ave somewhere between 2012 & 2014.  The most recent reroute didn't go all the way to US-12/45.

If anything, I would recommend the 'new' `+X02` point be removed, and replaced with the old one with a relabel, as that was where the reroute original happened on the eastern side.  
`*OldIL19 http://www.openstreetmap.org/?lat=41.952098&lon=-87.909336`

File: hwy_data/IL/usail/il.il019.wpt [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+ProAve http://www.openstreetmap.org/?lat=41.966540&lon=-87.998376
+WoodDaleRd http://www.openstreetmap.org/?lat=41.963381&lon=-87.978872
 IL83 http://www.openstreetmap.org/?lat=41.960862&lon=-87.959290
-+X01 http://www.openstreetmap.org/?lat=41.957416&lon=-87.915773
-+X02 http://www.openstreetmap.org/?lat=41.952277&lon=-87.909722
-+X03 http://www.openstreetmap.org/?lat=41.951416&lon=-87.897191
+YorkRd http://www.openstreetmap.org/?lat=41.960901&lon=-87.939564
+I-490 http://www.openstreetmap.org/?lat=41.960689&lon=-87.936882
++X01 http://www.openstreetmap.org/?lat=41.956148&lon=-87.932596
++X02 http://www.openstreetmap.org/?lat=41.954649&lon=-87.917741
```
Comment (rickmastfan67) [inline hwy_data/IL/usail/il.il019.wpt:17]: Change to the following, as this was the end point for the route relocation due to O'Hare expansion:
`*OldIL19 http://www.openstreetmap.org/?lat=41.952098&lon=-87.909336`

**Comment by rickmastfan67** [comment]
Also, original thread on this subject: https://forum.travelmapping.net/index.php?topic=5103.0

**Comment by Markkos1992** [comment]
I have no issues with what is being proposed by @rickmastfan67, but yes it is clear to me now that an updates entry is necessary.

**Comment by jteresco** [comment]
There was an IL 19 update committed and pushed (among several other changes Highway63 sent by email) this morning.  

https://github.com/TravelMapping/HighwayData/commit/15b8eb0d5864b6af826a2e7b133a9090714dfdbd

Closing this one.  Please send corrections if what I ended up committing and pushing is not the correct version. 

@rickmastfan67 @Markkos1992

**Comment by rickmastfan67** [comment]
@jteresco will the 'update' entry for IL-19 be pushed later?  It still needs one due to the reroute @ O'Hare.

**Comment by jteresco** [comment]
I don't have an updates entry for it because I don't know exactly what changed to the degree of needing such an entry.  I'd welcome one from someone who's followed the details more closely than I have.

I can easily enough hide the I-490 point (and will do so right now).

**Comment by rickmastfan67** [comment]
> I don't have an updates entry for it because I don't know exactly what changed to the degree of needing such an entry. I'd welcome one from someone who's followed the details more closely than I have.

@jteresco It was reroute between 'YorkRd' & '*OldIL19', so they could expand O'Hare.  Happened sometime between 2012 & 2014 per satellite imagery as I mentioned here: https://github.com/TravelMapping/HighwayData/pull/5945#issuecomment-1199933375

**Comment by rickmastfan67** [comment]
@jteresco We good here to destroy the branch now?  Didn't want to step on your feet here since you created it.

**Comment by jteresco** [comment]
Yes, I think this whole things has been resolved. @rickmastfan67

**Comment by rickmastfan67** [comment]
Deleted then.

**Pull request closed without merging**